Tryouts/Core/Abstractions/ISubscriptionClient.cs

```diff
@@ -1,27 +1,11 @@
-﻿/// ********************************************************************************************************
```

Comment:
Nitpick: please don't remove the license header

Reply:
restored